api/lib/application/memberships/membership-controller.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
   async update(request, h) {
     const membershipId = request.params.id;
     const userId = requestResponseUtils.extractUserIdFromRequest(request);
-    const membership = membershipSerializer.deserialize(request.payload);
+    const membership = await membershipSerializer.deserialize(request.payload);
```

Comment:
**question** 
Il est asynchrone le deserializer ? 😮

Reply:
A l'écriture du test unitaire, j'ai vu que je récupérais une promesse du deserialize, je veux bien voir avec vous sinon, peut être que mon test est mal écrit.

Reply:
vu ensemble => mauvais usage du `resolves` "Causes the stub to return a Promise which resolves to the provided value." https://sinonjs.org/releases/latest/stubs/
Utilisation du `returns`